docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -0,0 +1,197 @@
+- Focus - the user don't need to understand underlying concepts
+
+The downside is that only a limited set of features is available. Here, you can opt-out at any time by bringing your own collector setup. The current feature set focusses on providing the full configurability of backends integrated by OTLP. As a next step, meaningful filter options will be provided. Especially head and tail based sampling configurations.
+
+### Telemetry Operator
+The TracePipeline resource is managed by the Telemetry Operator, a typical Kubernetes operator responsible for managing the custom parts of the Otel Collector configuration.
+
+![Operator resources](./assets/tracing-resources.drawio.svg)
+
+The Telemetry Operator watches all TracePipeline resources and related Secrets. Whenever the configuration changes, it validates the configuration and generates a new configuration for the Otel Collector, where a ConfigMaps for the configuration is generated. Furthermore, referenced Secrets are copied into one Secret that is mounted to the Collector as well.
+Furthermore, the operator manages the full lifecycle of the Otel Collector Deployment itself. With that, it only gets deployed when there is an actual TracePipeline defined. At anytime you can opt-out of using the tracing feature by not specifying a TracePipeline.
+
+## Setting up a TracePipeline
+
+In the following a typical setup of a TracePipeline gets discussed. For a overview of all available attributes please have a look at the [reference document](./../../../05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md)
+
+### 1. Create a TracePipeline with an output
+1. To ship traces to a new OTLP output, create a resource file of kind TracePipeline:
+
+```yaml
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: TracePipeline
+metadata:
+  name: jaeger
+spec:
+  output:
+    otlp:
+      endpoint:
+        value: http://tracing-jaeger-collector.kyma-system.svc.cluster.local:4317
+```
+
+That will configure the underlying Otel Collector with a pipeline for traces. The receiver of the pipeline will be of type OTLP and be accessible via the `telemetry-otlp-traces` service. As exporter an `otlp` or an `otlphttp` exporter gets used, dependent on the configured protocol.
+
+2. To create the instance, apply the resource file in your cluster.
+    ```bash
+    kubectl apply -f path/to/my-trace-pipeline.yaml
+    ```
+
+3. Check that the status of the TracePipeline in your cluster is `Ready`:
+    ```bash
+    kubectl get logpipeline
+    NAME              STATUS    AGE
+    http-backend      Ready     44s
+    ```
+
+### 2. Switch the protocol to HTTP
+
+To use the HTTP protocol instead of the default GRPC, use the `protocol` attribute and assure that the proper port gets configured as part of the endpoint. Typically port 4317 is used for GRPC and port 4318 for HTTP.
```

Comment:
```suggestion
To use the HTTP protocol instead of the default GRPC, use the `protocol` attribute and ensure that the proper port is configured as part of the endpoint. Typically port `4317` is used for GRPC and port `4318` for HTTP.
```